plugin/kubernetes/object/informer.go

```diff
@@ -68,10 +70,25 @@ func DefaultProcessor(convert ToFunc, recordLatency RecordLatencyFunc) Processor
 						recordLatency(d.Object.(meta.Object))
 					}
 				}
+				cleanObj(d.Object)
 			}
 			return nil
 		}
 	}
 }
 
+func cleanObj(i interface{}) {
```

Comment:
I find it way more obvious if the endpoint cleanup happens in the To functions, this is too hidden

Reply:
If it happens in the To functions, the data is erased before it is needed when calculating latency after processing is done.  Currently, this happens, and it causes the latency metric to never report anything.

Reply:
no. cleaning up in the To functions is where this code belong, now it becomes hidden and we can only hope the switch will catch all different types